src/NUnitEngine/nunit.engine.tests/Runners/TestEngineRunnerTests.cs

```diff
@@ -39,7 +39,7 @@ namespace NUnit.Engine.Runners.Tests
     //[TestFixture(typeof(ProcessRunner))]
     [TestFixture(typeof(MultipleTestDomainRunner), 1)]
     [TestFixture(typeof(MultipleTestDomainRunner), 3)]
-    [TestFixture(typeof(MultipleTestProcessRunner), 1)]
+    //[TestFixture(typeof(MultipleTestProcessRunner), 1)]
```

Comment:
Starting a ProcessRunner of any kind has not been possible for a long time. Really these should be ignored.

I frankly have no idea why MultipleTestProcessRunner was working with the TestAgency included and ProcessRunner was not. MultipleTestProcessRunner actually creates ProcessRunners. This would be a good issue to create.